photutils/psf/epsf.py

```diff
@@ -261,6 +255,10 @@ class EPSFBuilder:
         have odd sizes along both axes to ensure a well-defined central
         pixel.
 
+    norm_radius : float, optional
+        The pixel radius over which the ePSF is normalized. If `None` or
```

Comment:
Maybe I'm missing something, but I think `None` does *not* work here?  Or if so I don't know where it's accounted for. But I also thin that's *good*, so I'd say just remove the option from the docstring...